app/javascript/packs/home/index/button_click.js

```diff
@@ -30,5 +30,6 @@ $(document).on("click", ".like", function() {
   console.log(likeButton);
   $(this)
     .toggleClass("color-blue")
-    .toggleClass("color-toggle");

```

Comment:
Convention is use the dot along with the first line:
```js
$(this).
    toggleClass("color-blue").
    toggleClass("color-toggle");
```

Reply:
It automatically changes when I save the file.